demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -67,31 +72,45 @@ def check_for_release_notes_valid_file_path(file_path):
 
         return file_path
 
-    def execute_update(self):
+    def handle_existing_rn_version_path(self, rn_path: str) -> str:
+        """ Checks whether the existing RN version path exists and return it's content.
+
+            :param rn_path: The rn path to check
+
+            :rtype: ``str``
+            :return
+                The content of the rn
+        """
+        if self.existing_rn_version_path:
+            self.should_delete_existing_rn = False if self.existing_rn_version_path == rn_path else True
+            try:
+                with open(self.existing_rn_version_path, 'r') as f:
+                    return f.read()
+            except Exception as e:
+                print_error(f'Failed to load the previous release notes file content: {e}')
+        return ''
+
+    def execute_update(self) -> bool:
+        """ Obtains the information needed in order to update the pack and executes the update.
+
+            :rtype: ``bool``
+            :return
+                Whether the RN was updated successfully or not
+        """
         if self.pack in IGNORED_PACK_NAMES:
             print_warning(f"Release notes are not required for the {self.pack} pack since this pack"
                           f" is not versioned.")
+            return False
         else:
```

Comment:
since you have a `return` statement in the condition, you can remove the use of `else`

Reply:
Fixed